### PR TITLE
feat: comp image

### PR DESCRIPTION
**Pull request by jsuryakt**
Please review responsive image

### CODE REVIEW

File: minimal.css [hunk 1]
```diff
@@ -439,3 +439,16 @@ code {
 .shadow {
   box-shadow: 2px 2px 10px 3px rgb(0, 0, 0, 0.3);
 }
+
+/* image  */
+
+.img-fluid {
```
Comment (sah-shah7) [inline minimal.css:445]: It will be better if each components have their own style sheet.

File: Components/image/image.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    minimal
+                    <h1 class="under">UI</h1>
+                </h1>
+            </header>
+        </a>
+        <button id="bar" class="menu-bar" onclick="menu()"><i class="bi bi-list"></i></button>
+    </div>
+
+    <div class="container">
+        <div class="sidebar content">
```
Comment (irajbhinde) [inline Components/image/image.html:27]: In future or if time permits now, you can use aside for side bars

File: Components/image/image.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            <a class="sidebar-link" href="/components/button/button.html">button</a>
+            <a class="sidebar-link" href="/components/card/card.html">card</a>
+            <a class="sidebar-link sidebar-active" href="/components/image/image.html">image</a>
+            <a class="sidebar-link" href="/components/input/input.html">input</a>
+            <a class="sidebar-link" href="/components/typography/typography.html">typography</a>
+            <a class="sidebar-link" href="/components/navigation/navigation.html">navigation</a>
+            <a class="sidebar-link" href="/components/slider/slider.html">slider</a>
+        </div>
+
+        <div class="main content">
```
Comment (irajbhinde) [inline Components/image/image.html:41]: You can use <main> tag here for the main unique content

File: minimal.css [hunk 4]
```diff
@@ -439,3 +439,16 @@ code {
 .shadow {
   box-shadow: 2px 2px 10px 3px rgb(0, 0, 0, 0.3);
 }
+
+/* image  */
```
Comment (irajbhinde) [inline minimal.css:443]: You can remove commented lines

**Review by irajbhinde — COMMENTED** [review]
Looks good. Few feedbacks
1) You are taking style from minimal.css, you can name the stylesheet as image.css
2) You can use appropriate HTML tags now / in future

**Pull request merged**